README.md

```diff
@@ -86,9 +71,49 @@ shapely
 fiona
 ```
 
-One of dependencies of `eo-learn-mask` subpackage is `lightgbm` package. On Windows it requires 64 bit Python distribution. If having problems during installation please check [LightGBM installation guide](https://lightgbm.readthedocs.io/en/latest/Installation-Guide.html).
+One of the `[MASK_EXTRA]` dependencies is the `lightgbm` package. On Windows it requires 64 bit Python distribution. If having problems during installation please check [LightGBM installation guide](https://lightgbm.readthedocs.io/en/latest/Installation-Guide.html).
+
+### PyPi distribution
+
+The package requires Python version **>=3.8** . If this is your first time installing `eo-learn`, or you aren't sure what kind of installation you require, then you can safely install it with:
+
+```bash
+pip install "eo-learn[ALL]"
+```
+
+To better control your dependencies, you can install just the core dependencies of `eo-learn` with:
 
-Some subpackages contain extension modules under `extra` subfolder. Those modules typically require additional package dependencies which don't get installed by default.
+```bash
+pip install eo-learn
+```
+
+Alternatively, you can decide for yourself which modules you want to be operational by providing the corresponding extras. E.g., if you are only interested in the `io` and `mask` modules, or in the extra dependencies of the `features` module, you can install them with:
+
+```bash
+pip install "eo-learn[IO, MASK]"
+pip install "eo-learn[FEATURES_EXTRA]"
+```
+
+Here is a full list of all the modules' dependencies and their extras:
+
+- `COREGISTRATION`
+- `FEATURES`
+- `FEATURES_EXTRA`
```

Comment:
perhaps "`FEATURES` and `FEATURES_EXTRA`" to conserve space? what do you think?

Reply:
agreed, done